docs/components/concepts/resource-deletion.md

```diff
@@ -0,0 +1,74 @@
+---
+id: resource-deletion
+title: "Resource deletion"
+description: "Use resource deletion to remove process definitions and decision requirements graphs from Camunda."
+---
+
+You can use resource deletion to remove resources from the Camunda Platform. There are two types of resources that can be deleted:
+
+1. Process definitions
+2. Decision Requirements Graphs (DRG)
+
+You can delete a resource using [Operate](../../components/operate/operate-introduction.md), or by sending
+the [delete resource command](/docs/apis-tools/grpc.md#deleteresource-rpc) to the Zeebe API.
+
+There are several reasons to delete resources from a cluster:
```

Comment:
I would likely move this section up after line 10, as before getting into how a user can delete a resource, they may want to know *why*.